### ISSUE TITLE
Conquest system math error

### ISSUE THREAD

**Issue by VincentHoshino** [DarkstarProject/darkstar#4752]
**_I have:_**

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:** 
30171223_0

**_Source Branch_** (master/stable) **:** 
stable

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
Conquest point gain/loss being calculated incorrectly with negative values

map\conquest_system.cpp

auto loss = std::min<int>(points * influences[i] / 5000 - influences[nation], influences[i]);
 should be  
auto loss = std::min<int>(points * influences[i] / (5000 - influences[nation]), influences[i]);
 
Issue discovered by Ranzera
